custom_components/govee_light_ble/config_flow.py

```diff
@@ -48,7 +48,7 @@ async def async_step_user(
             address = discovery_info.address
             if address in current_addresses or address in self._discovered_devices:
                 continue
-            if not discovery_info.name.startswith('Govee_'):
+            if not discovery_info.name.startswith('Govee_','ihoment_'):
```

Comment:
double brackets are needed to pass a tuple to startswith
(('Govee_', 'ihoment_'))

see: [https://www.w3schools.com/python/ref_string_startswith.asp](https://www.w3schools.com/python/ref_string_startswith.asp)